src/config/sample-data.js

```diff
@@ -0,0 +1,336 @@
+export const sampleData = {
+  appSessions: [{
```

Comment:
Can we make this a bit cleaner?

1. Create sampleData without the `day` properties
2. Loop through `appSessions` and `studioSessions`
3. Add the `day` property for each data point